package/yast2-printer.changes

```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Wed May 18 19:01:28 UTC 2016 - mchandras@suse.de
+
+- Remove /bin/ping dependency. All references to 'ping' have been
+  removed in 0ba65748c88b("apply jsmeix patches upstream"). This allows
```

Comment:
On this level we want to reference RPM versions, not git commits. I have checked that that commit was released in 3.1.2.